### PR TITLE
Made 3 different lang sections, added new words.

### LINKED ISSUE #71: Include swear words for moderation!!
Introduce as many swear words as possible each separated by a space. 
The file to be changed is:
`roBOT/blob/master/commands/scripts/moderate_words.txt`

Bengali and Hindi words are also welcome!

### PR DESCRIPTION
**Pull request by smart-worker**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #71 

Addresses #---

## Type of change

Please delete options that are not relevant.

- [ ] edited moderate_words.txt

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] build(3.8)
- [ ] build(3.9)

**Test Configuration**:
* OS:
* OS Version:

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Additional Details:

### CODE REVIEW

File: commands/scripts/moderate_words.txt [hunk 1]
```diff
@@ -1 +1,17 @@
-fuck motherfucker bitch
\ No newline at end of file
+English:
```
Comment (soham4abc) [inline commands/scripts/moderate_words.txt:1]: no need to mention the languages.... all words should be separated by a newline or space no colons or tab spaces

Comment (smart-worker) [reply commands/scripts/moderate_words.txt:1]: Ok ok got it.

**Review by soham4abc — CHANGES_REQUESTED** [review]
Don't use tab spaces or language tags

File: commands/scripts/moderate_words.txt [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-  kamine 
-  bsdk
-
-Bengali: 
-  suorer baccha
-  sala 
-  harami
-  
+fuck motherfucker bitch
+fuck for god's sake
```
Comment (soham4abc) [inline commands/scripts/moderate_words.txt:2]: remove

File: commands/scripts/moderate_words.txt [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
+fuck you
```
Comment (soham4abc) [inline commands/scripts/moderate_words.txt:3]: remove

File: commands/scripts/moderate_words.txt [hunk 4]
```diff
@@ -1,17 +1,11 @@
...
-
-Bengali: 
-  suorer baccha
-  sala 
-  harami
-  
+fuck motherfucker bitch
+fuck for god's sake
+fuck you  
+maa kasam
```
Comment (soham4abc) [inline commands/scripts/moderate_words.txt:4]: only kasam

File: commands/scripts/moderate_words.txt [hunk 5]
```diff
@@ -1,17 +1,11 @@
...
+chutiya
+gadhe
+kamine 
+bsdk 
+suorer baccha
```
Comment (soham4abc) [inline commands/scripts/moderate_words.txt:9]: only shuorer

**Review by soham4abc — CHANGES_REQUESTED** [review]

**Review by soham4abc — APPROVED** [review]
LGTM

**Pull request merged**